source/renderer/app/analytics/getCustomDimensions.ts

```diff
@@ -0,0 +1,35 @@
+import { Environment } from '../../../common/types/environment.types';
+import {
+  CPU_DIMENSION_KEY,
+  OS_DIMENSION_KEY,
+  RAM_DIMENSION_KEY,
+  USES_HARDWARE_WALLET_DIMENSION_KEY,
+  USES_LEGACY_WALLET_DIMENSION_KEY,
+  VERSION_DIMENSION_KEY,
+} from '../config/analyticsConfig';
+import { getShortCpuDescription } from '../utils/getShortCpuDescription';
+import { formattedBytesToSize } from '../utils/formatters';
+import AdaApi from '../api/api';
+
+const booleanToText = (flag: boolean) => (flag ? 'yes' : 'no');
+
+export const getCustomDimensions = async (
+  environment: Environment,
+  adaApi: AdaApi
+) => {
+  const usesByronWallets = (await adaApi.getWallets()).some(
```

Comment:
Small suggestion for single wallet request
```
const wallets = await adaApi.getWallets()
const usesHardwareWallets = wallets.some(
  (wallet) => wallet.isHardwareWallet
);
```

Reply:
Good catch, not sure why I missed it. Fixed.